package-lock.json

```diff
@@ -1,8 +1,6787 @@
 {
   "name": "@architect-io/cli",
   "version": "1.11.3-rc.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Changing the lockfile version seems to have blown up the lockfile 😬  I'm not familiar with the consequences of this change with regards to cli utilities. Are we okay to leave this change or should we revert it?

Reply:
Node 15+ uses npm 7+ which changed the lockfile format. Harmless, but annoying diff. I'll upgrade our ci jobs to use node 16 so we are in sync.

Reply:
https://github.com/architect-team/architect-cli/commit/bc1f8a7df649e80863219672389b92a595a9af2f